web/frontend/angular/src/app/main-pages/register/register.component.ts

```diff
@@ -24,6 +25,7 @@ export class RegisterComponent implements OnInit {
         this.authForm = this.formBuilder.group({
             email: ["", Validators.required],
```

Comment:
An email is not required. It is optional. Only the username, password, and password confirmation are required. Does this form even have a username field?

Reply:
@alinaah I see you replaced the email with a username field. However, the form should have _both_ and email field and a username field. To be clear, the form should have the following fields:

* username (required)
* email (optional)
* password (required)
* confirm password (required)